fili-core/src/main/java/com/yahoo/bard/webservice/util/SimplifiedIntervalList.java

```diff
@@ -369,4 +370,22 @@ public SimplifiedIntervalList subtract(SimplifiedIntervalList that) {
                 .collect(Collectors.toList());
         return IteratorUtils.chainedIterator(periodIterators);
     }
+
+    /**
+     * Returns an Optional of inner interval with the oldest starting instance.
+     *
+     * @return the Optional of inner interval with the oldest starting instance
+     */
+    public Optional<Interval> getStart() {
+        return stream().min(IntervalStartComparator.INSTANCE);
+    }
```

Comment:
simplified interval list guarantees that the start of the earliest interval will be the earliest start.